src/server/accounting.c

```diff
@@ -2259,7 +2260,14 @@ void log_alter_records_for_attrs(job *pjob, svrattrl *plist) {
 			svrattrl *svrattrl_list = NULL;
 			job_attr_def[i].at_encode(&pjob->ji_wattr[i], &phead, job_attr_def[i].at_name, NULL, ATR_ENCODE_CLIENT, &svrattrl_list);
 			for (cur_plist = plist; cur_plist != NULL; cur_plist = (svrattrl *)GET_NEXT(cur_plist->al_link)) {
-				if (strcmp(cur_plist->al_name, job_attr_def[i].at_name) != 0)
+				int j;
+				int found = 0;
```

Comment:
This is really a nit pick- is it possible to change the variable name to "ignore" instead of found?